src/main/java/uk/ac/bath/se/SplashScreen.java

```diff
@@ -7,6 +7,19 @@
 
 class SplashScreen {
 
+    //this stArea is for saving the user name created by the player and past it to the DungeonView.
+    // so the game can save the historical records
+
+    static String stArea;
+
+    public void setstArea(String stArea) {
+        this.stArea = stArea;
+    }
+
+    public static String getstArea(String stArea) {
```

Comment:
the object is not instantiated the stArea string has no value so static not needed

Reply:
Yeah. I can do it.